kolena/workflow/model.py

```diff
@@ -203,11 +228,10 @@ def _populate_from_other(self, other: "Model") -> None:
             self.name = other.name
             self.metadata = other.metadata
             self.tags = other.tags
-            self.workflow = other.workflow
```

Comment:
Why?

Reply:
https://github.com/kolenaIO/kolena/pull/315#discussion_r1397890819

Reply:
It seems to me that [only workflow is meant to be a class attribute](https://github.com/kolenaIO/kolena/blob/trunk/kolena/workflow/define_workflow.py#L84) and [the other class attributes](https://github.com/kolenaIO/kolena/blob/trunk/kolena/workflow/model.py#L73-L87) should be moved to instance attributes only

Reply:
Going to revert this diff for this PR and suggest in a new one